vhdbuilder/packer/windows-image.env

```diff
@@ -22,17 +22,17 @@ WINDOWS_2022_GEN2_BASE_IMAGE_VERSION=20348.2762.241006
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:23h2-datacenter-core:latest
 WINDOWS_23H2_BASE_IMAGE_SKU=23h2-datacenter-core
-WINDOWS_23H2_BASE_IMAGE_VERSION=25398.1189.241004
+WINDOWS_23H2_BASE_IMAGE_VERSION=25398.1251.241105
 
 # CLI example to get all available image version under a SKU (suffix g2 for Gen 2):
 #   az vm image list --all --publisher MicrosoftWindowsServer --offer WindowsServer --output table -s 23h2-datacenter-core-g2
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:23h2-datacenter-core-g2:latest
 # NOTE: Please update $global:patchUrls and $global:patchIDs in vhdbuilder/packer/generate-windows-vhd-configuration.ps1 on demand
 WINDOWS_23H2_GEN2_BASE_IMAGE_SKU=23h2-datacenter-core-g2
-WINDOWS_23H2_GEN2_BASE_IMAGE_VERSION=25398.1189.241004
+WINDOWS_23H2_GEN2_BASE_IMAGE_VERSION=25398.1251.241105
 
 # Please uncomment the following lines and set a larger os disk size that is at least 30GB when your PR check-in fails
-# WINDOWS_2019_CONTAINERD_OS_DISK_SIZE_GB=30
+WINDOWS_2019_CONTAINERD_OS_DISK_SIZE_GB=32
```

Comment:
We should not update this for production. Instead, we should remove some old cached images and packages.